gidgethub/apps.py

```diff
@@ -1,10 +1,14 @@
 """Support for GitHub Actions."""
-from typing import cast, Any, Dict
+
+from __future__ import annotations
+
+from typing import cast, Any, Dict, TYPE_CHECKING
 
 import time
 import jwt
 
-from gidgethub.abc import GitHubAPI
+if TYPE_CHECKING:
```

Comment:
This perhaps should be done as a separate PR.

Reply:
Sure I can do that. Let me know and I'll set up another PR that can be merged before this one. The `TYPE_CHECKING` guard is needed to prevent a circular import since `abc` now imports from `apps` in this PR.